src/components/GameItem.js

```diff
@@ -16,10 +24,38 @@ class GameItem extends Component {
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    let icon;
+    if (this.props.itemType === "rock") {
+      icon = ItemIcons.rock;
+    }
+
+    if (this.props.itemType === "litter") {
+      icon = ItemIcons.litter;
+    }
+
+    if (this.props.itemType === "bush") {
+      icon = ItemIcons.bush;
+    }
+
+    if (this.props.itemType === "flower") {
+      icon = ItemIcons.flower;
+    }
+
+    if (this.props.itemType === "mushroom") {
+      icon = ItemIcons.mushroom;
+    }
+
+    let litterSpottedClass;
+    if (this.props.isLitterSpotted === true) {
+      litterSpottedClass = "game-item spotted-litter";
+    } else if (this.props.isLitterSpotted === false) {
+      litterSpottedClass = "game-item spotted-nature";
+    } else {
+      litterSpottedClass = "game-item";
+    }
```

Comment:
Consider refactoring your logic so it doesn't rely on `this.props.isLitterSpotted` being either `true`, `false`, or `null`